objection_engine/anim.py

```diff
@@ -521,23 +506,25 @@ def comments_to_scene(comments: List[CommentBridge], name_music = "PWR", **kwarg
         polarity = analizer.get_sentiment(comment.body)
 
         # Calculate how to split up comment text for line wrapping
-        tokens = nlp(comment.body)
-        sentences = [sent.text.strip() for sent in tokens.sents]
-        joined_sentences = []
-        i = 0
-        while i < len(sentences):
-            sentence = sentences[i]
-            if len(sentence) > 85: # Long sentences should be wrapped to multiple shorter lines
-                text_chunks = [chunk for chunk in wrap(sentence, 85)]
-                joined_sentences = [*joined_sentences, *text_chunks]
-                i += 1
-            else:
-                if i + 1 < len(sentences) and len(f"{sentence} {sentences[i+1]}") <= 85: # Maybe we can join two different sentences
-                    joined_sentences.append(sentence + " " + sentences[i+1])
-                    i += 2
-                else:
-                    joined_sentences.append(sentence)
-                    i += 1
+        # tokens = nlp(comment.body)
+        # sentences = [sent.text.strip() for sent in tokens.sents]
```

Comment:
Hi!

I'd prefer removing code rather than commenting it unless there's a really good reason to comment it out. Git history is always available at the end of the day

Reply:
I totally agree! I should've deleted this code before I made the pull request, guess I just forgot. 😅 Will go ahead and commit those changes as soon as I can!